src/components/ReactTags.js

```diff
@@ -131,13 +131,16 @@ class ReactTags extends Component {
       return this.props.handleFilterSuggestions(query, suggestions);
     }
 
+    const queryIndex = (item) =>
```

Comment:
Add a spec under https://github.com/prakhar1989/react-tags/blob/75fd6ad6a6f9c36586b20cf1afd02fa3b2bca053/__tests__/reactTags.test.js#L421 describe block to check when multiple strings are rendered using partial search, example you can add one more suggestion `orange` in the suggestion list and search using `an` so that `banana` and `orange` both are shown i suggestions.

Reply:
If i will revert previous samples from "Or" to "Ea" it will already shows partial search with result "Pear" and "Peach", isn't it?
Is it still necessary to add more suggestion and test?

Reply:
@ad1992 thank you for your feedback. See my comments for requested changes.